app.js

```diff
@@ -1,8 +1,54 @@
 // import functions and grab DOM elements
-
+const disImage0 = document.getElementById('cup-0');
+const disImage1 = document.getElementById('cup-1');
+const disImage2 = document.getElementById('cup-2');
+const winsEl = document.getElementById('wins');
+const lossesEl = document.getElementById('losses');
+const totalEl = document.getElementById('total');
+const button0 = document.getElementById('button-0');
+const button1 = document.getElementById('button-1');
+const button2 = document.getElementById('button-2');
 // let state
+let wins = 0;
+let total = 0;
+
+disImage0.src = 'assets/cup.png';
+disImage1.src = 'assets/cup.png';
+disImage2.src = 'assets/cup.png';
+
+// create handler function for user guess
+function handleGuess(id) {
```

Comment:
We want the handleGuess funciton to to take 2 parameters-- handleGuess(userGuess, correctSpot)